acqa-repo1-aws-tf12-part1.tf

```diff
@@ -617,16 +645,18 @@ resource "aws_cloudfront_distribution" "acqa-test-cloudfront1" {
     # Must use us-east-1 for your Certs (certificate manager)
     acm_certificate_arn = "arn:aws:acm:us-east-1:641885301384:certificate/b5b12158-c4cd-4662-bc04-ecfe18a1bdc3"
   }
+
+  web_acl_id = "<web_acl_id>"
 }
 
 # Codecommit
 resource "aws_codecommit_repository" "acqa-test-ccrepo1" {
   repository_name = "acqa-test-ccrepo1"
   description     = "acqa-test-ccrepo1"
   tags = {
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: Ensure scan on push is enabled on AWS ECR repository**
 
**Vulnerability**
Unscanned images may contain vulnerabilities.
 
**Remediation**
Enable scan_on_push in image_scanning_configuration. Amazon ECR image scanning helps in identifying software vulnerabilities in your container images.

source: accurics-